### ISSUE TITLE
OVS CNI support

### ISSUE THREAD

**Issue by brysk** [cni-genie/CNI-Genie#194]
Hi Genie Folks,

Can you please let me know if OVS CNI is supported?
https://github.com/kubevirt/ovs-cni

**Comment by haibinxie** [comment]
Yes, CNI-Genie is not customized to any specific plugin.  If the plugin is compliant with CNI spec, integration with CNI Genie is natural.

**Comment by brysk** [comment]
thanks @haibinxie 

Is CNI Genie still under development? I can see latest commit is from Jul 17, 2019. 

I have also faced issues when installing Genie. DaemonSet is under apps/v1 now, not extensions.
https://raw.githubusercontent.com/cni-genie/CNI-Genie/master/conf/1.8/genie-plugin.yaml

Is it possible to have the following interface configuration inside the pod:

eth0 OVS CNI
eth1  SRIOV CNI
eth2 kube pod network (flannel, calico or something else)
